en/simulation/gazebo.md

```diff
@@ -197,19 +206,29 @@ For more information see: [Simulation > Run Simulation Faster than Realtime](../
 Joystick and thumb-joystick support are supported through *QGroundControl* ([setup instructions here](../simulation/README.md#joystickgamepad-integration)).
 
 
-### Simulating GPS Noise
+### Improving Distance Sensor Performance.
```

Comment:
Note to me. Remove fullstop.

@julianoes FYI, this is the old "Change World" section. Name was confusing.